pb/lmsysmd/load/model/v1/model.proto

```diff
@@ -12,12 +12,10 @@ service ModelService {
 }
 
 message BatchCreateModelRequest {
-  string google_sheet_url = 1 [
+  bytes csv_file = 1 [
```

Comment:
Why not keep the previous field? Google sheets tracks revisions. Recommend for you to track revisions https://developers.google.com/drive/api/guides/manage-revisions.

Reply:
Logging offline conversation here - misunderstanding. Will be using google sheets url in order to track revision history ids on each upload.